.github/workflows/publish-docker-image.yml

```
@@ -25,4 +25,7 @@ jobs:
         with:
           context: .
           push: true
-          tags: ${{ secrets.DOCKER_USERNAME }}/cligram:latest
\ No newline at end of file
+          tags: ${{ secrets.DOCKER_USERNAME }}/cligram:latest
+          build-args: |
+            TELEGRAM_API_ID=${{ secrets.TELEGRAM_API_ID }}
+            TELEGRAM_API_HASH=${{ secrets.TELEGRAM_API_HASH }}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)